pkg/util/requestbuffers.go

```diff
@@ -5,6 +5,8 @@ import (
 	"bytes"
 )
 
+const maxInPoolRequestBufferSize = 1024 * 1024 // 1MB
```

Comment:
how do we know which value is 'the right value'?

Reply:
good point. we don't know, although in my opinion, I understand that `1MB` should be generous enough to cover a wide spectrum of use cases.

Reply:
i'd describe that in a comment

Reply:
_added @ bc47fd2b405ff4b0ddb1899a4151d3ed5676994b_